openapi-mock/Dockerfile

```diff
@@ -1,12 +1,20 @@
-FROM node:11-alpine
+FROM node:11-alpine as builder
 RUN apk --no-cache upgrade

```

Comment:
The update of the packages mainly has to happen in the productive image, not the build image. To assure known security vulnerabilities in the used packages are fixed.